lib/dynflow/director.rb

```diff
@@ -163,6 +164,9 @@ def handle_event(event)
       execution_plan_manager = @execution_plan_managers[event.execution_plan_id]
       if execution_plan_manager
         execution_plan_manager.event(event)
+      elsif event.optional
```

Comment:
nit code improvement idea: Would it be possible to always reject the event and let the event itself decide how to handle that?
Meaning having resolve listener and on reject eighter raise or do nothing depending on the optional flag.
Entirely optional, just throwing ideas :)

Reply:
It would probably be more elegant, but let's keep it simple for now.

Reply:
> Entirely _optional_
:laughing: